### PR TITLE
configuration for yarn package-win

### PR DESCRIPTION
**Pull request by wuyuyue**

### CODE REVIEW

**Comment by wuyuyue** [comment]
we use electron-builder, I find we could config on seperate file "electron-builder.yml" instead of package.json(success in windows); the detail as the format https://github.com/Azure/BatchExplorer/blob/master/electron-builder.yml

**Comment by ashchan** [comment]
> we use electron-builder, I find we could config on seperate file "electron-builder.yml" instead of package.json(success in windows); the detail as the format https://github.com/Azure/BatchExplorer/blob/master/electron-builder.yml

This looks cool.

File: packages/electron-app/src/main.ts [hunk 1]
```diff
@@ -8,15 +8,15 @@ let mainWindow: Electron.BrowserWindow | null
...
+  PROD: `file://${path.join(__dirname, '../../react-app/build/index.html')}`,
 }
 
 function createWindow() {
   // Create the browser window.
   mainWindow = new BrowserWindow({
     width: 800,
     height: 600,
-    icon: `${__dirname}/../build/icon.icns`,
+    // icon: `${__dirname}/../build/icon.icns`,
```
Comment (ashchan) [inline packages/electron-app/src/main.ts]: Instead of commenting this out, we can just delete it.

File: electron-builder.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+appId: com.nervos.neuron
```
Comment (ashchan) [inline electron-builder.yml]: There's still an `electron-builder.yml` file in the root folder. Was this supposed to be deleted?

File: packages/electron-app/src/main.ts (hunk 1 above)
Comment (ashchan) [reply packages/electron-app/src/main.ts]: Oh I see. That icon file was removed (perhaps by accident) from this commit:

https://github.com/nervosnetwork/neuron/commit/3a36b6c103bd2179d0667451f18adfa30886faea#diff-de61d9af2b61d0fbe27ceb0ee8a5f555

@Keith-CY We had that file in `build` folder with the purpose to bundle that for the Mac app. Could you take a look at that and see if we can bring it back? (maybe we should not ignore the `build` folder?)

**Pull request merged**